atrium-api/src/types/integer.rs

```diff
@@ -53,6 +59,14 @@ macro_rules! uint {
         pub struct $lim_nz<const MAX: $primitive>($nz);
 
         impl<const MAX: $primitive> $lim_nz<MAX> {
+            /// The smallest value that can be represented by this limited non-zero
+            /// integer type.
+            pub const MIN: Self = Self($nz::MIN);
+
+            /// The largest value that can be represented by this limited non-zero integer
+            /// type.
+            pub const MAX: Self = Self(unsafe { $nz::new_unchecked(MAX) });
```

Comment:
The `unsafe` blocks are necessary because `Option::unwrap` can't be used in a `const` context yet on stable Rust (otherwise we could use `$nz::new(MAX).unwrap()`).